CHANGELOG.md

```diff
@@ -17,12 +21,6 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 - Fix template helpers issue converting port number for the webhook endpoint.
 
-## [v0.8.2] - 2020-06-02
-
-### Added
-
-- Added dates to CHANELOG entries.
-
```

Comment:
FYI Absence of this entry was misleading, that the release was not present, so existing v0.8.2 git tag got updated by mitake in attempt to make next release. This changelog entry was added to avoid any confusion, even though change made in the release is not so much customer relevant.

Reply:
Fixed it with [new commit](https://github.com/giantswarm/kong-app/pull/60/commits/b3a0e76f025bee238bf81cdccafd86bac71e9bd2)